src/System.Management.Automation/engine/remoting/commands/StartJob.cs

```diff
@@ -512,10 +512,12 @@ public virtual Version PSVersion
 
             set
             {
-                RemotingCommandUtil.CheckPSVersion(value);
-
-                // Check if specified version of PowerShell is installed
-                RemotingCommandUtil.CheckIfPowerShellVersionIsInstalled(value);
+                // PSVersion value can only be 5.1 for Start-Job.
+                if (!(value.Major == 5 && value.Minor == 1))
```

Comment:
Do we need to check if this is Windows and return a different error if used on Linux/macOS?